.ci/validation.sh

```diff
@@ -1365,30 +1365,26 @@ spotless)
   ./mvnw -e --no-transfer-progress clean spotless:check -P spotless-autofix
   ;;
 
-openrewrite-recipes)
-  echo "Cloning and building OpenRewrite recipes..."
-  PROJECT_ROOT="$(pwd)"
+sanity-check)
   export MAVEN_OPTS="-Xmx4g -Xms2g"
-
+  echo "Sanity Check 🦢"
+  echo "Cloning and building OpenRewrite recipes 📥"
   cd /tmp
   git clone https://github.com/checkstyle/checkstyle-openrewrite-recipes.git
   cd checkstyle-openrewrite-recipes
   mvn -e --no-transfer-progress clean install -DskipTests
-
-  cd "$PROJECT_ROOT"
```

Comment:
why this coupling on `PROJECT_ROOT`? 

If we can inline, then it seems obsolete.